python/kfserving/docs/KFServingClient.md

```diff
@@ -16,11 +16,66 @@ The APIs for KFServingClient are as following:
 
 Class | Method |  Description
 ------------ | ------------- | -------------
+KFServingClient | [create_creds](#create_creds) | Create Credentials|
 KFServingClient | [create](#create) | Create KFService|
 KFServingClient | [get](#get)    | Get the specified KFService|
 KFServingClient | [patch](#patch)  | Patch the specified KFService|
 KFServingClient | [delete](#delete) | Delete the specified KFService |
 
+## create_creds
+> create_creds(self, platform, namespace=None, **kwargs)
+
+Create `Secret` and `Service Account` for AWS and GCP accordingly to credentials information. Once the `Service Account` ceated, user can configure the `Service Account` in the [V1alpha1ModelSpec](V1alpha1ModelSpec.md) for kfserving.
+
+The API returns name of created `Service Account`.
+
+### Example
+
+Example for creating AWS credentials.
+```python
+KFServingClient
+from kfserving import KFServingClient
+
+KFServing = KFServingClient()
+KFServing.create_creds(platform='AWS',
```

Comment:
can also add support for azure? for reference see:
https://github.com/kubeflow/kubeflow/pull/2676

Reply:
@rakelkar Yes, that's good to support azure, but here I'm wonder the azure is supported by kfserving? from [here](https://github.com/kubeflow/kfserving/blob/5305cd5c377b7494e0ee27a84cd4c78ef7f3f892/pkg/controller/kfservice/resources/credentials/service_account_credentials.go#L109) seems only support AWS and GCP. if so we need to support in the kfserving firstly? Correct me if any mistake. Thanks.